wandb/sdk/wandb_init.py

```diff
@@ -648,22 +648,8 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
             run.config.update({k: v}, allow_val_change=True)
 
         self.backend = backend
-        module.set_global(
```

Comment:
YAS!!

Reply:
I think we should move it to `Run.__new__` but didn't want to do it in this PR